src/main/resources/db/changelog/db.changelog-insert-role-permission.xml

```diff
@@ -0,0 +1,45 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet id="1.1.0" author="volodymyr.protsiv">
```

Comment:
Can we have ```sql``` file with these data for insert?

Reply:
Added role_permission.sql. 
In db.changelog-insert-role-permission.xml I used an sqlFile tag to execute insert statments.